### PR TITLE
create conda env file

### PR DESCRIPTION
**Pull request by gideonite**
This is the first step towards making this code reproducible for the future. The environment file contains a lot of cruft, e.g. higher order deps that are probably brittle.

### CODE REVIEW

File: projects/super_res/environment.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  - kornia
+  - accelerate
+  - xarray
+  - seaborn
+  - zarr
+  - google-cloud-sdk
+  - fsspec
+  - gcsfs
+  - joblib
+  - tensorboard
```
Comment (cynicalsavant) [inline projects/super_res/environment.yaml:28]: I think we can remove tensorboard since I'm using wandb

File: projects/super_res/access_gcp.py
Comment (cynicalsavant) [inline projects/super_res/access_gcp.py:1]: What seems to be the purpose of this file?

File: projects/super_res/environment.yaml (hunk 1 above)
Comment (cynicalsavant) [reply projects/super_res/environment.yaml:28]: Also, could you run python trainer.py without any issues?

File: projects/super_res/access_gcp.py
Comment (gideonite) [reply projects/super_res/access_gcp.py:1]: My fault, this was part of my experiment which is no longer needed. I just removed it. Apologies for the confusion.

File: projects/super_res/environment.yaml (hunk 1 above)
Comment (gideonite) [reply projects/super_res/environment.yaml:28]: Right, good catch. Meanwhile, `wandb` isn't there.

**Pull request merged**